README.md

```diff
@@ -94,6 +94,23 @@ There is isn't much to contribute to, but still if you want to suggest a new alg
 First Create an Issue and state your contribution.If approved you are welcome to send a PR. 
 Suggestions are welcome on how to further increase the accuracy of the model.
 
+#### Fork the Repo 
+  - In the top-right corner of the page, click Fork.
+#### Create a Branch
+```sh
+git checkout -b <branch name>
+``` 
+#### Push Changes to Master Repo
```

Comment:
This is really good. But I would want you to add something regarding addressing issues.

Just mention that contributors when opening a PR must address the issue they are addressing to.

Reply:
And just one more thing @gabrielcerteza . 
It would be better if you could create a seperate CONTRIBUTING.md file and copy-paste this there.
Thanks again for the contribution 👍

Reply:
What would you like me to name the separate CONTRIBUTING.md file?

Reply:
CONTRIBUTING.md  only